packages/sdk-rtl/package.json

```
@@ -1,6 +1,6 @@
 {
   "name": "@looker/sdk-rtl",
-  "version": "21.0.19",
+  "version": "21.0.20",
```

yep, release-please will handle this for us (based on the commit prefix of `fix:`, although doing this won't mess up release-please since it looks at `.release-please-manifest.json` as the source of truth and just overwrites this version number accordingly)

```suggestion
  "version": "21.0.19",
```